backend/cmd/migrate/migrations/000008_topology_cache_add_composite_key.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE topology_cache DROP CONSTRAINT topology_cache_id_key;
```

Comment:
shouldn't this be?
```suggestion
ALTER TABLE topology_cache DROP CONSTRAINT topology_cache_id_pkey;
```

Reply:
This is not a primary key its just a UNIQUE key so its not a PK

Reply:
let me get the output of all they keys real quick

Reply:
This is what it looks like right now (testing locally)

```
clutch=# \d topology_cache
                          Table "public.topology_cache"
      Column       |            Type             | Collation | Nullable | Default
-------------------+-----------------------------+-----------+----------+---------
 id                | character varying           |           |          |
 data              | jsonb                       |           |          |
 resolver_type_url | character varying           |           |          |
 metadata          | jsonb                       |           |          |
 updated_at        | timestamp without time zone |           |          | now()
Indexes:
    "topology_cache_id_key" UNIQUE CONSTRAINT, btree (id)
    "topology_cache_id_search_idx" btree (id varchar_pattern_ops)
    "topology_cache_metadata_idx" gin (metadata jsonb_path_ops)```

Reply:
ah i forgot you could even create tables without PKs